test/helpers.ts

```diff
@@ -13,7 +13,7 @@ export function useEnvironment(fixtureProjectName: string) {
   beforeEach("Loading hardhat environment", function () {
     process.chdir(path.join(__dirname, "fixture-projects", fixtureProjectName));
     process.env.HARDHAT_NETWORK = networkName;
-    this.hre = require("hardhat");
+    this.hre = require(networkName);
```

Comment:
I'm pretty sure this is a mistake, you always want to require the `"hardhat"` module here.

Reply:
Fixed via https://github.com/pcaversaccio/xdeployer/pull/208/commits/be943ed1e52139bab618297c4eef0ef810de6f5b